main.py

```diff
@@ -1,7 +1,62 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+
+    Actor.objects.create(
+        first_name="George",
+        last_name="Klooney",
+    )
+    Actor.objects.create(
+        first_name="Kianu",
+        last_name="Reaves",
+    )
+    Actor.objects.create(
+        first_name="Scarlett",
+        last_name="Keegan",
+    )
+    Actor.objects.create(
+        first_name="Will",
+        last_name="Smith",
+    )
+    Actor.objects.create(
+        first_name="Jaden",
+        last_name="Smith",
+    )
+    Actor.objects.create(
+        first_name="Scarlett",
+        last_name="Johansson",
+    )
```

Comment:
Create a list of tuples with first name and last name. After that iterate through this list and use unpacking in for loop to create objects to avoid code duplication